discovery-provider/src/api/v1/users.py

```diff
@@ -494,6 +495,48 @@ def get(self, user_id):
         return success_response(tracks)
 
 
+history_route_parser = reqparse.RequestParser()
+history_route_parser.add_argument("user_id", required=False, type=str)
+history_route_parser.add_argument("limit", required=False, type=int)
+history_route_parser.add_argument("offset", required=False, type=int)
+history_response = make_full_response(
+    "history_response_full", full_ns, fields.List(fields.Nested(activity_model_full))
+)
+
+
+@full_ns.route("/<string:user_id>/history/tracks")
+class TrackHistoryFull(Resource):
```

Comment:
just note to self this is not permissioned (like other routes here & historically how this has been).

Reply:
What does that mean?

Reply:
im just writing this for the future reader (myself) that this route doesn't require you to be authenticated (e.g. history is public, not private)